lib/syslog-names.h

```diff
@@ -36,6 +36,9 @@
 /* extract priority */
 #define LOG_PRI(p)  ((p) & LOG_PRIMASK)
 
+#define SEVERITY_CODE(n)    ((n) & 7)
```

Comment:
These are not really needed to be visible, could we keep them local to the original compilation unit ?